backend/package.json

```diff
@@ -3,12 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "cross-env NODE_ENV=test LOG=true npx nodemon ./src/bin/www",
+    "start": "npm run db-init && cross-env NODE_ENV=test LOG=true npx nodemon ./src/bin/www",
     "lint": "npx eslint src/**/*.js tests/**/*.js --fix",
     "test:lint": "npx eslint src/**/*.js tests/**/*.js --fix",
-    "test:src": "cross-env NODE_ENV=test npx nyc mocha --file tests/utils/*.js --file tests/routes/*.js --timeout 10000",
+    "test:src": "npm run db-init && cross-env NODE_ENV=test npx nyc mocha --file tests/utils/*.js --file tests/routes/*.js --timeout 10000",
```

Comment:
curious -- is cross-env not persisting across this and the above call?

Reply:
i'm not sure - didn't try it without..
looking at the cross-env docs it seems like you need to use `cross-env-shell` to have environment variables persist through an entire shell command ... maybe?